examples/Model.elm

```diff
@@ -418,23 +416,22 @@ recordResource =
 
 decodeRecord : Decode.Decoder Record
 decodeRecord =
-    (Decode.map4 Record
+    Decode.map4 Record
         (Decode.field "id" Decode.string)
         (Decode.maybe (Decode.field "title" Decode.string))
         (Decode.maybe (Decode.field "description" Decode.string))
         (Decode.field "last_modified" Decode.int)
-    )
 
 
 
 -- Kinto API calls
 
 
 fetchRecord : Maybe Kinto.Client -> RecordId -> Cmd Msg
-fetchRecord client recordId =
-    case client of
-        Just client ->
-            client
+fetchRecord clientInstance recordId =
+    case clientInstance of
+        Just c ->
+            c
```

Comment:
It might make more sense to have:
```elm
fetchRecord maybeClient recordId =
    case maybeClient of
        Just client ->
            client
```